geomstats/geometry/product_riemannian_metric.py

```diff
@@ -98,7 +98,9 @@ def is_intrinsic(self, point):
         intrinsic: bool
             Whether intrinsic coordinates are used for all manifolds.
         """
-        assert self.default_point_type == 'vector'
+        if self.default_point_type != 'vector':
```

Comment:
in fact if we reach that point there really is a problem, it's not that it was not implemented

Reply:
Modified!